packages/waku/src/lib/middleware/rsc/ssr.ts

```diff
@@ -319,7 +319,9 @@ export const renderHtml = async <Context>(
               if (command === 'dev') {
                 const filePath = normalizePath(
                   file.startsWith('@fs/')
-                    ? file.slice('@fs/'.length)
+                    // FIXME This is ugly. We need to refactor it.
+                    // remove '@fs'(3) on Unix and '@fs/'(4) on Windows
+                    ? file.slice(2 + path.sep.length)
```

Comment:
I just found that `@fs/User/path/to/file` on Unix, but `@fs/C://path/to/file` on Windows

Is this kind of **feature** on Vite?